### ISSUE TITLE
Edit "How to add a link to a file" KB article

### ISSUE THREAD

**Issue by EWashb** [department-of-veterans-affairs/va.gov-cms#10730]
## Background
There are a existing KB articles relating to PDFs, however editors still are often confused about how to correctly link to a PDF.  This causes broken links and frustration for the editors.

By reviewing and editing the existing articles, we hope to provide the editors with clearer directions for using PDFs within the CMS

### Existing KB articles

- [How to add a link to a file](https://prod.cms.va.gov/help/cms-basics/file-link)
- [How to troubleshoot broken form downloads/Bad PDF Links from Find a Form](https://prod.cms.va.gov/help/va-forms/how-to-troubleshoot-broken-form-downloadsbad-pdf-links-from-find-a-form) (not positive that this article is relevant to this topic but it mentioned PDFs so I wanted to add it here just in case)

Duplicate ticket: #10448
- Fold in these changes too

## Acceptance Criteria

- [x] Existing KB article(s) are reviewed and updated to provide clearer, more accurate directions for editors
- [x] Article reviewed by CMS team
- [x] Updated KB article is published

**Comment by stefaniefgray** [comment]
This is the current "How to PDF" KB article: https://prod.cms.va.gov/help/cms-basics/file-link

@EWashb Do you think this could be updated/revised or does the subject deserve a new article?

It's the first result when searching the Knowledge Base for "PDF" but the title could definitely be clearer as well as the steps in the process

Added "Knowledge Base" GitHub ticket label and assigned myself

**Comment by EWashb** [comment]
@stefaniefgray possibly! I was thinking this article could be more of the dos and don'ts of PDFing. We could possibly repurpose the other one to be more succient. I'm open to suggestions here!

**Comment by laflannery** [comment]
Just had a meeting with @TroyA6 and @stefaniefgray We think it would be best to edit this article rather than create a new one.

I edited the Title/Description of this ticket to reflect that

**Comment by TroyA6** [comment]
@stefaniefgray @laflannery - I have revised [the document](https://docs.google.com/document/d/1_h8HHKUsmAowo_j6Dcul3vIbSX2Iin_NDLV24Tz_pZo/edit) in MS WORD (just for editing) specifically to reflect Links to PDF files... Please take a look and make suggestions, comments, etc. Thanks!

**Comment by laflannery** [comment]
@TroyA6 I didn't edit your article, but instead I made [a new one here](https://docs.google.com/document/d/140Cj7k8v9disL0Vc7p3NwDMcj_3xABr9mJE7r7LM4yM/edit#heading=h.jcquwxnu6cfc) I didn't want to remove what you have here (I actually used some of it) but my proposed updates are different than yours:

- Italics should not be used per the VA style guide
- Fields and actions are just in bold, we don't need the quotes (again that's a style guide direction)
- I also had a tough time between PDF vs file.  I know we are trying to solve our PDF problem but editors can upload other types of files, so I don't want to make this a PDF only article.  I sprinkled PDF into the beginning of the article hoping this will help, and I'm hoping the other changes help too but overall I tried to stay away from only saying PDF.
- This article is about adding links for files - I feel that we should really stick to that, I think it makes it clearer, which is why I removed everything about other fields or editing
- We can't put screenshots in between ordered lists - it breaks the lists and therefore accessibility. I'm also not sure it's entirely needed here? I think the instructions are straightforward enough - now that we have cleaned them up and having too many screenshots will make this harder to keep current if/when things change.
- I also edited the entire article because there were also things that needed to be updated in those sections and also to keep the entire article consistent, so I have the whole thing here.

Let me know what you think or if you want to chat at all about this. Thanks!
